proposals/sandbox/krustlet.md

```diff
@@ -0,0 +1,92 @@
+# Krustlet
+
+## Description
+
+Krustlet is a Kubernetes Kubelet implementation for running WebAssembly (Wasm) modules on nodes.
+Krustlet acts as a Kubelet by listening on the event stream for new pods that the scheduler assigns
+to it based on specific Kubernetes tolerations. It also can be extended to enable other types of
+non-Wasm runtimes using a "Provider Model" similar to [Virtual
+Kubelet](https://github.com/virtual-kubelet/virtual-kubelet), but its primary use case is for Wasm.
+
+## Project Goal
+
+The primary goal of Krustlet is to be the easiest way to integrate Wasm support as seamlessly as
+possible into Kubernetes. It will allow for users to run heterogeneous clusters where both
+containers and Wasm modules can run (though not necessarily on the same node)
+
+## Current Status
+
+Krustlet is very close to a 1.0 release. It was created in late 2019 and has grown along with the
+rest of the Wasm community. Essentially 1.0 is stating that that basic features (but not all
+advanced features such as pod resources, full networking stack integration, health checks) for a
+kubelet are in place and the software is reliable enough to use in production. Continued growth of
+the project will include implementing more of the advanced kubelet features and tracking upstream
+changes to the the WebAssembly Systems Interface spec (WASI).
+
+## Value for the CNCF ecosystem
+
+Krustlet is a great fit for the CNCF ecosystem. First and foremost, it is a Kubernetes-related
+project. That in and of itself is not enough to justify inclusion, but Krustlet has proven to be an
+interesting alternative runtime for Kubernetes and has driven interest in the wider Kubernetes
+community around using Wasm. As Sandbox projects are meant to "[e]ncourage public visibility of
```

Comment:
wasm and other non-container workloads?

Reply:
Yeah, that would probably be good to say as well